cola/widgets/diff.py

```diff
@@ -250,6 +250,22 @@ def set_highlighted(self, line_number):
         """Set the line to highlight"""
         self.highlight_line = line_number
 
+    def newfile_line(self):
+        if self.lines and self.highlight_line >= 0:
+            # Find the next valid line
+            for i in range(self.highlight_line, len(self.lines)):
+                # take the "new" line number: last value in tuple
+                line_number = self.lines[i][-1]
+                if line_number > 0:
+                    return line_number
+
+            # Find the previous valid line
+            for i in range(self.highlight_line - 1, -1, -1):
+                # take the "new" line number: last value in tuple
+                line_number = self.lines[i][-1]
+                if line_number > 0:
+                    return line_number
```

Comment:
It seems like the end of this function should probably have `return 0` as a fallback, or maybe `return None` is more appropriate so that it avoids the `line_numbers` code path in the Edit command.